gcloud/datastore/test_client.py

```diff
@@ -18,29 +18,44 @@
 class TestClient(unittest2.TestCase):
 
     DATASET_ID = 'DATASET'
+    CONNECTION = object()
```

Comment:
I have avoided doing this, but think it's a good idea. I was on the fence if your testing manifesto agreed with or frowned up creating a test default for non-default arguments (i.e. ones with `None` as their default).